### PR TITLE
Support react-native 0.60+ and bug fixes

### PR DESCRIPTION
**Pull request by thanhcuong1990**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -2,25 +2,23 @@
...
 android {
-    compileSdkVersion 23
-    buildToolsVersion "25.0.0"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion 18
```
Comment (ttkien) [inline android/build.gradle]: Why do we need to increase minSdkVersion?

Comment (thanhcuong1990) [reply android/build.gradle]: @ttkien no need to increase. just rollback it.

**Comment by luco** [comment]
@alwx Please, merge.

**Comment by alwx** [comment]
@thanhcuong1990 thanks for your contribution and sorry that it took me so much time to merge it :)

File: ios/RNPhotoView.m [hunk 2]
```diff
@@ -153,7 +154,7 @@ - (CGFloat)initialZoomScaleWithMinScale {
         if (ABS(boundsAR - imageAR) < 0.17) {
             zoomScale = MAX(xScale, yScale);
             // Ensure we don't zoom in or out too far, just in case
-            zoomScale = MIN(MAX(self.minimumZoomScale, zoomScale), self.maximumZoomScale);
+            zoomScale = MIN(MAX(minZoom, zoomScale), minZoom);
```
Comment (namvoeh) [inline ios/RNPhotoView.m:157]: @thanhcuong1990 Given zoom scale range is `[0.1, 3]`
if the zoom scale is `2`, `MIN(MAX(minZoom, zoomScale), minZoom)` returns `0.1`

can you share the reason for this change?

**Comment by jsamr** [comment]
@alwx Would you mind releasing on NPM? The latest release is 3 years old...
image

**Pull request merged**